tycho-its/projects/compiler.exclude/pom.xml

```diff
@@ -12,7 +12,7 @@
 			<plugin>
 				<groupId>org.eclipse.tycho</groupId>
 				<artifactId>tycho-maven-plugin</artifactId>
-				<version>${tycho.version}</version>
+				<version>${tycho-version}</version>
```

Comment:
Just wondering: Is this test actually executed in the CI?

Reply:
good question `tycho-its/src/test/java/org/eclipse/tycho/test/compiler/CompilerExclude.java`
do we match test name patterns?

Reply:
`CompilerExclude` must be `CompilerExcludeTest` to be executed.

Reply:
will do a follow up pr

Reply:
If you like you can also include it here.

Reply:
done. lets see what jenkins says

Reply:
The test fails, I think we have two options of your choice:

1. Take a look why it fails
2. Add an Ignore and create an issue to investigate the failure later on

Reply:
@laeubi i fixed the test. do we know when the license check will work again?